pypi/build.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/sh
+
```

Comment:
```suggestion
set -e -o pipefail
```
Then you don't need the `if test $? -ne 0; then exit 1; fi` checks anymore.

Reply:
Good point. Change made